### DISCUSSION TITLE
How to downgrade the gauge-js version to any specific version?

### DISCUSSION THREAD

**Discussion by Ansuman92** [getgauge/gauge#1873] [General]
Hello folks, I'm just trying to downgrade the gauge js version from 2.3.15 to 2.3.12 but when I ran the cmd 'gauge install js --version 2.3.12' in my terminal then the version doesn't seems to be changes back to 2.3.12
Could anyone please let me know how to downgrade the gauge-js version?
NOTE: I'm using gauge version 1.1.4.

**Comment by sriv** [comment]
Gauge will pick up the latest compatible version of the runner that is installed in the machine. So in your case if you have gauge-js `2.3.12` and `2.3.15` both installed, Gauge will prefer the latter.

Try uninstalling newer versions, ex: `gauge uninstall js --version 2.3.15`

**Reply by Ansuman92** [reply]
Thanks a lot for the quick response. I tried above step and matched the js and json-report version and hence it worked well.

**Reply by sriv** [reply]
Cool. Out of curiosity, why do you need to downgrade? We encourage the use of latest versions.

**Reply by Ansuman92** [reply]
Actually I was using 2.3.15 only but getting 'Failed to read Prototype EOF' error after running any scripts at the end of output irrespective of whether any script pass or fails and since my teammates were using js 2.3.12 and running fine for them so I thought of downgrading my js and json-report plugin and later it worked fine.
